alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -196,20 +225,32 @@ transTx utxo tx =
     fee = txfee' tbody
     forge = mint' tbody
     interval = vldt' tbody
+    vkkeyset = txwitsVKey' (wits' tx)
```

Comment:
This needs a scary FIXME: whatever we do it can't be this (see discussion about signature stripping)

Reply:
@TimSheard we can pass plutus the `reqSignerHashes` field of the transaction once #2198 is merged.

Reply:
It is now fixed.